README.md

```diff
@@ -42,15 +42,9 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+- GitHub: [@githubhandle](https://github.com/29td)
+- Twitter: [@twitterhandle](https://twitter.com/NonhlanhlaMnde6)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 ## 🤝 Contributing
 
```

Comment:
I recommend you change the `Acknowledgments` section to actually reflect some acknowledgments. You can thank Microverse or say this was a Microverse project.